client/src/App.js

```diff
@@ -32,6 +33,10 @@ const App: ComponentType<any> = useDragDrop(() => (
         exact
         index
       />
+      <Route
+        element={<SsoLogin />}
+        path='sso-login'
```

Comment:
Let's use an underscore in the path to be consistent with other multi-word routes.